### PR TITLE
bugfix/v1-100/fixed learning page styles

### PR DESCRIPTION
**Pull request by Julia-Maliutsina**
Fixed according to QA:
1. Video container, it's preview, play icon.
2. Start the test button.
3. Comments label display for Safari.

### CODE REVIEW

**Review by irina-sinitsyna — DISMISSED** [review]

File: frontend/src/pages/LearningCourse/styled.ts [hunk 1]
```diff
@@ -238,6 +247,7 @@ export const StartTestButton = styled(Button)({
   boxShadow: 'none!important',
   [theme.breakpoints.down('md')]: {
     fontSize: '16px!important',
+    padding: '0 !important',
```
Comment (k-krasilnikova) [inline frontend/src/pages/LearningCourse/styled.ts:250]: Remove all the "important" marks

File: frontend/src/pages/LearningCourse/styled.ts [hunk 2]
```diff
@@ -275,7 +286,7 @@ export const ToggleDescription = styled(Typography)({
   fontWeight: '500',
   color: '#131313',
   [theme.breakpoints.up('md')]: {
-    display: 'none',
+    display: 'none !important',
```
Comment (k-krasilnikova) [inline frontend/src/pages/LearningCourse/styled.ts:289]: !

**Review by k-krasilnikova — CHANGES_REQUESTED** [review]

File: frontend/src/pages/LearningCourse/styled.ts (hunk 2 above)
Comment (Julia-Maliutsina) [reply frontend/src/pages/LearningCourse/styled.ts]: Removed

File: frontend/src/pages/LearningCourse/styled.ts (hunk 1 above)
Comment (Julia-Maliutsina) [reply frontend/src/pages/LearningCourse/styled.ts]: Removed

**Pull request merged**